src/xml.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/xml"
+	log "github.com/sirupsen/logrus"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+// JUnitTestSuites is a set of mysqltest suite.
+type JUnitTestSuites struct {
```

Comment:
How about renaming it to `XUnitTestSuite`? "JUnit" is a testing framework in Java, it's weird to name it here.

Reply:
Thanks for the suggestion about renaming!

It is done. PTAL.